contributed_definitions/nyaml/NXenergydispersion.yaml

```diff
@@ -34,8 +38,19 @@ NXenergydispersion(NXobject):
     doc: |
       Diameter of the dispersive orbit
   energy_scan_mode(NX_CHAR):
-    doc: |
+    doc:
+    - |
       Way of scanning the energy axis (fixed or sweep).
+    - |
+      xref:
+        spec: ISO 18115-1:2023
+        term: 12.65
+        url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.65
+    - |
+      xref:
+        spec: ISO 18115-1:2023
+        term: 12.66
+        url: https://www.iso.org/obp/ui/en/#iso:std:iso:18115:-1:ed-3:v1:en:term:12.66
```

Comment:
This is what we maybe still wanted to extend with other modes, e.g. transmission modes, jittered mode etc., no?

Reply:
Discussion in https://github.com/FAIRmat-NFDI/nexus_definitions/pull/127